rkt/cp.go

```diff
@@ -0,0 +1,271 @@
+// Copyright 2017 The rkt Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/appc/spec/schema/types"
+	"github.com/hashicorp/errwrap"
+	"github.com/rkt/rkt/common"
+	pkgPod "github.com/rkt/rkt/pkg/pod"
+	"github.com/spf13/cobra"
+)
+
+var (
+	cmdCp = &cobra.Command{
+		Use:   "cp [--app=APPNAME] [UUID:]SRC_PATH [UUID]:DEST_DIR_PATH",
+		Short: "Copy a file/directory to/from an app within a rkt pod",
+		Long: `UUID should be the UUID of a running pod.
+
+UUID should be specified once, either on the SRC_PATH or the DEST_DIR_PATH.`,
+		Run: ensureSuperuser(runWrapper(runCp)),
+	}
+	flagCpAppName string
+)
+
+func init() {
+	cmdRkt.AddCommand(cmdCp)
+	cmdCp.Flags().StringVar(&flagCpAppName, "app", "", "name of the app to copy to/from within the specified pod")
+}
+
+func runCp(cmd *cobra.Command, args []string) (exit int) {
+	if len(args) < 2 {
+		cmd.Usage()
+		return 255
```

Comment:
nit, 254 for consistency.

Same for the other return values.

In this case it might make more sense to not use that for everything (since it can't be confused with application exit), but the error message should be enough to distinguish these error cases